src/interface/variable_pack.hpp

```diff
@@ -181,11 +255,16 @@ VariableFluxPack<T> MakeFluxPack(const vpack_types::VarList<T> &vars,
           std::pair<std::string, IndexPair>(v->label(), IndexPair(vstart, vindex - 1)));
     }
   }
+  if (!(sparse_name == "")) {
```

Comment:
This is more preference but I would replace lines like these with. 
```
if ( not sparse_name.empty() ) 
```
I think it is a little bit more readable.